include/gatekeeper_net.h

```diff
@@ -38,11 +37,20 @@ struct gatekeeper_if {
 	/* Name of the interface. Needed for setting/getting bonded port. */
 	const char	*name;
 
+	/* Timeout for cache entries (in seconds) for Link Layer Support. */
+	uint32_t	arp_cache_timeout;
```

Comment:
Renaming this field to `arp_cache_timeout_sec` makes it clear that the unit is seconds everywhere it's used.